bower.json

```diff
@@ -23,6 +23,7 @@
     "datatables": "~1.10.12",
     "chosen": "~1.5.1",
     "fontawesome": "~4.6.3",
-    "leaflet": "^0.7.7"
+    "leaflet": "^0.7.7",
+    "moment": "~2.17.1"
```

Comment:
Yeah, I knew. However, i would like to ask where will the script be used on the site?